File: temp/src/Player.java
public class Player {

    private final String name;
    private final Mark mark;

    public Player(String name, Mark mark) {
        this.name = name;
        this.mark = mark;
    }

    public String getName() {
        return this.name;
    }

    public Mark getMark() {
        return this.mark;
    }

    public boolean makeMove(Board board, int row, int col)  {
        try {
            board.setCell(row, col, this.mark);
            return true;
        } catch (Exception e) {
            return false;
        }
    }
}

File: tic_tac_py/src/player.py
class Player:
    def __init__(self, name, mark):
        self.name = name
        self.mark = mark

    def get_name(self):
        return self.name

    def get_mark(self):
        return self.mark

    def make_move(self, board, row, col):
        try:
            board.set_cell(row, col, self.mark)
            return True
        except Exception:
            return False
